websites/A/ArcViewer/presence.ts

```diff
@@ -0,0 +1,173 @@
+	"90Degree" = "https://i.imgur.com/S1JazD5.png",
+	LawlessEasy = "https://i.imgur.com/g89WIhy.png",
+	LawlessExpert = "https://i.imgur.com/MwJO8XO.png",
+	LawlessExpertPlus = "https://i.imgur.com/2Pkfbsz.png",
+	LawlessHard = "https://i.imgur.com/zq7N05l.png",
+	LawlessNormal = "https://i.imgur.com/Sm9lHhc.png",
+	Lawless = "https://i.imgur.com/0S2j43F.png",
+	LightshowEasy = "https://i.imgur.com/PdQg0hR.png",
+	LightshowExpert = "https://i.imgur.com/XoOh1hC.png",
+	LightshowExpertPlus = "https://i.imgur.com/1CEVUT6.png",
+	LightshowHard = "https://i.imgur.com/ltthw6g.png",
+	LightshowNormal = "https://i.imgur.com/AkFJbtq.png",
+	Lightshow = "https://i.imgur.com/tttOnGz.png",
+	LegacyEasy = "https://i.imgur.com/KvYCJE8.png",
+	LegacyExpert = "https://i.imgur.com/GEEPOJ4.png",
+	LegacyExpertPlus = "https://i.imgur.com/Di0s2HS.png",
+	LegacyHard = "https://i.imgur.com/X8gYxVN.png",
+	LegacyNormal = "https://i.imgur.com/cbfykqi.png",
+	Legacy = "https://i.imgur.com/1Lstuag.png",
+	NoArrowsEasy = "https://i.imgur.com/paIHyy8.png",
+	NoArrowsExpert = "https://i.imgur.com/bDKWwVA.png",
+	NoArrowsExpertPlus = "https://i.imgur.com/2qeKbua.png",
+	NoArrowsHard = "https://i.imgur.com/a6CVhdU.png",
+	NoArrowsNormal = "https://i.imgur.com/Uw6Ci9A.png",
+	NoArrows = "https://i.imgur.com/gM9e7R1.png",
+	OneSaberEasy = "https://i.imgur.com/3JR9H4i.png",
+	OneSaberExpert = "https://i.imgur.com/OpdqBoF.png",
+	OneSaberExpertPlus = "https://i.imgur.com/3FgTB6G.png",
+	OneSaberHard = "https://i.imgur.com/VTLKvXF.png",
+	OneSaberNormal = "https://i.imgur.com/6AhE9Oy.png",
+	OneSaber = "https://i.imgur.com/a8pREAV.png",
+	StandardEasy = "https://i.imgur.com/7jE3gMV.png",
+	StandardExpert = "https://i.imgur.com/c5lpTfm.png",
+	StandardExpertPlus = "https://i.imgur.com/aZyA5kb.png",
+	StandardHard = "https://i.imgur.com/41pumvg.png",
+	StandardNormal = "https://i.imgur.com/FqAg2k7.png",
+	Standard = "https://i.imgur.com/u3Ou4UR.png",
+	Easy = "https://i.imgur.com/fNND69T.png",
+	Expert = "https://i.imgur.com/Zy5kVyl.png",
+	ExpertPlus = "https://i.imgur.com/yxPfNmQ.png",
+	Hard = "https://i.imgur.com/cSexHo3.png",
+	Normal = "https://i.imgur.com/uWBt3mb.png",
+	UnknownEasy = "https://i.imgur.com/X2g8nos.png",
+	UnknownExpert = "https://i.imgur.com/tvdwov1.png",
+	UnknownExpertPlus = "https://i.imgur.com/tDsfuAG.png",
+	UnknownHard = "https://i.imgur.com/0jTbumO.png",
+	UnknownNormal = "https://i.imgur.com/1vbCA1u.png",
+	Unknown = "https://i.imgur.com/B7aVigy.png",
+}
+
+presence.on("UpdateData", async () => {
+	const [time, buttons, cover, mapSmallImages] = await Promise.all([
+			presence.getSetting<boolean>("time"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<number>("mapSmallImages"),
+		]),
+		presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		logs = await presence.getLogs();
+
+	for (const line of logs) {
```

Comment:
Make sure to merge/rebase the latest main into your branch.

Either:
* Add type check for string
* Assume strings and use string generic with `getLogs`